Sources/Testing/Testing.docc/TemporaryGettingStarted.md

```diff
@@ -119,18 +90,57 @@ export TOOLCHAINS=swift
 
 In Xcode, open the **Xcode** menu, then the Toolchains submenu, and select the
 development toolchain from the list of toolchains presented to you&mdash;it will
-<!--be presented with a name such as "Swift Development Toolchain 2023-01-01 (a)".-->
-be presented with a name such as "Swift 5.10 Development Snapshot 2023-01-01 (a)".
+be presented with a name such as "Swift Development Toolchain 2023-01-01 (a)".
 
 ### Running tests
 
+Navigate to the directory containing your package and run the following command:
+
+```sh
+swift test --enable-experimental-swift-testing
+```
+
+Swift Package Manager will build and run a test target that uses the testing
+library as well as a separate target that uses XCTest. To only run tests written
+using the testing library, pass `--disable-xctest` as an additional argument to
+the `swift test` command.
+
+#### Swift 5.10
+
+As of Swift 5.10, the testing library is not integrated into Swift Package
+Manager, but a temporary mechanism is provided for developers who want to start
+using it with their Swift packages and the Swift 5.10 toolchain.
+
+- Warning: This functionality is provided temporarily to aid in integrating the
+testing library with existing tools such as Swift Package Manager. It will be
+removed in a future release.
+
+To use the testing library with Swift 5.10, add a new Swift source
+file to your package's test target named "Scaffolding.swift". Add the following
+code to it:
+
+```swift
+import XCTest
+import Testing
+
+final class AllTests: XCTestCase {
+  func testAll() async {
+    await XCTestScaffold.runAllTests(hostedBy: self)
+  }
+}
+```
+
 Navigate to the directory containing your package, and either run `swift test`
 from the command line or click the Product&nbsp;&rarr;&nbsp;Test menu item in
-Xcode. You're done!
+Xcode.
 
 Tests will run embedded in an `XCTestCase`-based test function named
 `testAll()`. If a test fails, `testAll()` will report the failure as its own.
 
+#### Swift 5.9 or earlier
+
+The testing library does not support Swift 5.9 or earlier.
```

Comment:
If this is the final decision and we do not intend to support Swift 5.9 (Although it compiles fine now with Swift 5.9)

Should we also update `Package.swift`'s version to 5.9?

Reply:
Well we'd probably update it to 5.11 or later, but we're not quite there yet and we know some folks (including you!) want to continue to experiment.

But yes, we will eventually need to update our package version. Want to file an issue for that?